app/services/rag/ragas_evaluation.py

```
@@ -0,0 +1,477 @@
+                mode="recall", claim_decomposition_prompt=build_claim_decomposition_prompt()
+            ),
+        ],
+        llm=build_judge_llm(),
+    )
+
+    rows = scores.to_pandas().to_dict(orient="records")
+    if rows:
+        _warn_missing_columns(rows[0])
+    return {
+        result.case.id: _pick_scores(row)
+        for result, row in zip(results, rows)
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)